packages/plugins/src/cli/propose/index.ts

```diff
@@ -106,8 +108,15 @@ export const buildParsedConfigArray: BuildParsedConfigArray = async (
       process.exit(1)
     }
 
+    // Check that the system contracts are deployed.
     const provider = new SphinxJsonRpcProvider(rpcUrl)
-    await ensureSphinxAndGnosisSafeDeployed(provider)
+    // This auto-generated wallet is only used for internal testing purposes.
+    const wallet = new ethers.Wallet(getSphinxWalletPrivateKey(0), provider)
```

Comment:
We pass a wallet into the `ensureSphinxAndGnosisSafeDeployed` function in every case to simplify its logic